### ISSUE TITLE
[Accessibility][Screen reader - Property Pane_Ask a question]: Screen reader does not announce the state change information when the checkbox("Always prompt") gets selected or unselected.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2075]
**User Experience** 
If a screen reader will not provide any state change information about the checkbox selection  then screen reader dependent users will never get to know about the control property.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and turn on narrator.
2. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
3. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
4. Navigate to property pane appeared at right side of the screen and select "Always prompt" checkbox.
5. Observer the issue.

**Actual:** 
Screen reader does not announce the state change information when the checkbox("Always prompt") gets selected or unselected. Screen reader just announces the key pressed name as "Space"

**Expected:**
Screen reader should announce the state change information when the checkbox("Always prompt") gets selected or unselected. 

**MAS Impacted:** [MAS4.2.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={216e5b88-a1f8-425c-83e8-11bc4e4d74be})

**Attachment:**
[MAS4.2.2_Checkbox.zip](https://github.com/microsoft/BotFramework-Composer/files/4249279/MAS4.2.2_Checkbox.zip)
![MAS4 2 2_Checkbox](https://user-images.githubusercontent.com/60913131/75236046-3bc23f00-57e3-11ea-90fa-5ef02f8019cd.png)

**Comment by beyackle** [comment]
Tested this out and I'm not able to repro it. The SR reads "Space; checked" and "Space; unchecked" according to the key that got hit and the checkbox's new state.

**Comment by ashish315** [comment]
As checked the UI has been updated, now prompt has dropdown. So closing the bug.
